tools/install.sh

```diff
@@ -1,9 +1,22 @@
 #!/bin/bash
 set -e
 ## This script installs the Terra CLI from an unarchived release directory.
-## Dependencies: docker, gcloud
+## TERRA_CLI_DOCKER_MODE environment variable controls docker support. Set to
+##     SUPPORT_DOCKER (default) or LOCAL_ONLY to skip docker steps.
```

Comment:
Comment doesn't match the `DOCKER_NOT_AVAILABLE`, `DOCKER_AVAILABLE` literals below, or the default value.